alfredo-zimperz/precourse/console-bingo/bingo.js

```diff
@@ -0,0 +1,279 @@
+const CARD_LENGTH = 15
+const POINTS = 100
+
```

Comment:
lo que comentamos ayer de los espacios tras for, function, if, else, etc.... y recuerda primero constantes y despues variables

Reply:
Corregido, gracias!